src/coreComponents/mesh/FaceManager.cpp

```diff
@@ -201,7 +201,9 @@ void FaceManager::sortAllFaceNodes( NodeManager const & nodeManager,
     // The face should be connected to at least one element.
     if( facesToElements( faceIndex, 0 ) < 0 && facesToElements( faceIndex, 1 ) < 0 )
     {
-      GEOS_ERROR( "Face " << faceIndex << " is not connected to an element." );
+      GEOS_ERROR( getDataContext() << ": Face " << faceIndex << " is not connected to an element." <<
+                  "The cellBlocks of the CellElementRegions might not have referenced this face." <<
+                  " Be sure to include every primitive of every part of the mesh in the cellBlocks." );
```

Comment:
I am unsure of the readability of this message to the user.
Isn't this the error that we have when we forget to register a CellType in the input XML?